src/lib.rs

```diff
@@ -40,7 +40,7 @@
 //! - `-vvv` show debug
 //! - `-vvvv` show trace
 //!
-//! You can also customize the default logging level:
+//! You can also customize the default logging level with the `Verbosity<T>` generic:
```

Comment:
I am not a native english speaker, but this sounds a bit weird to me. @epage?

Reply:
It reads weird to me as well as this as well
- I'd personally refer to it as a "generic parameter"
- This puts emphasis on the type but calls it a generic parameter

I'm also not too sure what a better way of wording this is that specifically calls out the generic parameter in this way.  The only change I was thinking of was "You can customize the default logging level using `Verbosity`s generic parameter:"

Reply:
updated to suggestion